infra/base-images/base-builder/detect_repo.py

```diff
@@ -63,15 +75,34 @@ def get_repo(repo_path):
   Returns:
     The repo location or None
   """
-  output, return_code = execute(
-      ['git', 'config', '--get', 'remote.origin.url'],
-      location=repo_path,
-      check_result=True)
+  output, return_code = execute(['git', 'config', '--get', 'remote.origin.url'],
+                                location=repo_path,
+                                check_result=True)
   if return_code == 0 and output:
     return output.rstrip()
   return None
 
 
+def check_for_repo_name(repo_path, repo_name):
+  """Check to see if a github ref exists in a remote repository.
+
+  Args:
+    repo_path: The directory where the selected git repo exists
+    repo_name: The name of the target git repo
+
+  """
+  # Check if valid git repo.
+  print("Repo path: " + repo_path)
+  if not os.path.exists(os.path.join(repo_path, '.git')):
+    return False
+
+  out, _ = execute(['git', 'config', '--get', 'remote.origin.url'],
+                   location=repo_path)
+  out = out.split('/')[-1].replace('.git', '').rstrip()
+  print("Repo possability: " + out)
```

Comment:
Did you mean "possibility" or "possibly"?
I think "Possible repo: " is better than either.
I think this week we should replace most of these adhoc prints with python's logger.

Reply:
Done.